tests/handlers/external_test.py

```diff
@@ -19,15 +18,12 @@
     V1PodStatus,
     V1Volume,
 )
+from safir.testing.kubernetes import MockKubernetesApi
 
-from tests.support.constants import TEST_HOSTNAME
-from tests.support.kubernetes import assert_kubernetes_objects_are
+from moneypenny.models import Dossier
 
-if TYPE_CHECKING:
-    from httpx import AsyncClient
-
-    from moneypenny.models import Dossier
-    from tests.support.kubernetes import MockKubernetesApi
+from ..support.constants import TEST_HOSTNAME
+from ..support.kubernetes import MockKubernetesWatch
```

Comment:
Yeah, it should eventually but I've been holding off lifting things up into Safir until there are two clients of it so that I have some confirmation I got the API right. I want to add a proper test to the watcher in Gafaelfawr as well, so when that happens there will be two users and it will make sense to move it into Safir. (Or of course if your use of it happens first.)